Expensify/App
# fixed slow emoji entry & Keyboard focus

Please review @jasperhuangg 

### Details
1. Updated the web only value update statement to cross-platform `setNativeProps`.
2. Workaround for the Keyboard with `setTimeout`. It seems there is not another way for now.
3. Remove unnecessary state updates on Composer.
4. Fix Promise rejection error on Picker for native devices.

### Fixed Issues

Fixes #2920

### Tests | QA Steps
1. Open e.cash.
2. Open any chat.
3. Click the emoji Picker icon.
4. Select any emoji.
5. Emoji should be added immediately and there should not be a delay after Picker is closed.
6. Keyboard should open after Picker is closed.

### Tested On


### Screenshots


#### Web


https://user-images.githubusercontent.com/24370807/120968774-ea935000-c786-11eb-886b-9298e4802610.mp4


#### Mobile Web


#### Desktop


https://user-images.githubusercontent.com/24370807/120969031-4cec5080-c787-11eb-9ca6-9f920ff87721.mp4


#### iOS


#### Android


https://user-images.githubusercontent.com/24370807/120968938-2e865500-c787-11eb-8233-47139f85b171.mp4
